stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/images/AttachmentGalleryActivity.kt

```diff
@@ -44,7 +45,7 @@ public class AttachmentGalleryActivity : AppCompatActivity() {
             DateUtils.FORMAT_ABBREV_RELATIVE
         )
             .toString()
-            .decapitalize(resources.configuration.locales.get(0))
+            .decapitalize(Locale.getDefault())
```

Comment:
This would also be equivalent, as it uses the default Locale internally:

```suggestion
            .decapitalize()
```